systemd-user/icyci@.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=icyCI
```

Comment:
The regular service file includes:
`After=network-online.target mnt-resource.mount`
Is that not needed here?

Reply:
- `network-online.target` not defined for user instances (an interesting question if user instance liveness guarantees networking but it's marginal even here),
- `mnt-resource.mount` corresponds to `/mnt/resource`  -- not needed in my case but perhaps in any (system) case (I thought it was a leftover from your environment).

Reply:
> * `network-online.target` not defined for user instances (an interesting question if user instance liveness guarantees networking but it's marginal even here),

Interesting, okay. In that case icyci will likely need to improve when it comes to coping with network outages.

>     * `mnt-resource.mount` corresponds to `/mnt/resource`  -- not needed in my case but perhaps in any (system) case (I thought it was a leftover from your environment).

Oops, that's a scrap that was left from when I was running jobs on some Azure VMs. I'll remove it, thanks.